### PR TITLE
Adds objective logging to antags

### PR DESCRIPTION
**Pull request by ghost**
Title.

:cl:  
rscadd: Added antag objective logging
imageadd: added an image of me after working on this shit for like 6 hours and then Ling came on and fucking fixed it in two seconds goddamnit
/:cl:

### CODE REVIEW

**Comment by ThatLing** [comment]
Won't put it back

**Comment by ghost** [comment]
I'LL PUT IT BACK JEESH

**Comment by ThatLing** [comment]
You didn't put it back

File: code/modules/events/abductor.dm [hunk 1]
```diff
@@ -24,9 +24,8 @@
 	if(T.team_number > ABDUCTOR_MAX_TEAMS)
 		return MAP_ERROR
 
-	log_game("[key_name(scientist)] has been selected as [T.name] abductor scientist.")
-	log_game("[key_name(agent)] has been selected as [T.name] abductor agent.")
-
```
Comment (ThatLing) [inline code/modules/events/abductor.dm]: You removed this newline

File: code/game/gamemodes/devil/devil_game_mode.dm [hunk 2]
```diff
@@ -40,8 +40,8 @@
 		devils += devil
 		devil.special_role = traitor_name
 		devil.restricted_roles = restricted_jobs
-
-		log_game("[key_name(devil)] has been selected as a [traitor_name]")
+
```
Comment (ThatLing) [inline code/game/gamemodes/devil/devil_game_mode.dm]: Remove this tabulation

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 3]
```diff
@@ -30,7 +30,8 @@
 			pre_nukeops += new_op
 			new_op.assigned_role = "Nuclear Operative"
 			new_op.special_role = "Nuclear Operative"
-			log_game("[key_name(new_op)] has been selected as a nuclear operative")
+			//log_game("[key_name(new_op)] has been selected as a nuclear operative") | yogs - redundant
+
```
Comment (ThatLing) [inline code/game/gamemodes/nuclear/nuclear.dm]: Remove this newline

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Review by ThatLing — APPROVED** [review]

**Comment by DLCabbose** [comment]
nerd

**Pull request merged**